backoffice/src/test/java/com/wedriveu/backoffice/booking/BookingListGenerator.java

```diff
@@ -0,0 +1,43 @@
+package com.wedriveu.backoffice.booking;
+
+import com.wedriveu.services.shared.model.Booking;
+import com.wedriveu.shared.util.Position;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * @author Stefano Bernagozzi
+ */
+public class BookingListGenerator {
+    public static List<Booking> getBookingsFromBookingService() {
```

Comment:
Also here, where is all the PPS course knowledge about design patterns & code? Here i miss javadoc and it seems like a `Factory` pattern but you have to implement it right. And please constants instead of hard-coded strings/numbers